pkg/kvstore/allocator/allocator.go

```diff
@@ -538,7 +538,13 @@ func (a *Allocator) Allocate(ctx context.Context, key AllocatorKey) (idpool.ID,
 		k     = key.GetKey()
 	)
 
-	kvstore.Trace("Allocating key", nil, logrus.Fields{fieldKey: key})
+	log.WithField(fieldKey, key).Info("Allocating key")
```

Comment:
The same here is for this Info message to be changed to something else.

Reply:
Discussed on Slack, Info is fine